os/include/debug.h

```diff
@@ -404,6 +404,30 @@ Once LOGM is approved, each module should have its own index
 #define nllvdbg(...)
 #endif
 
+#ifdef CONFIG_DEBUG_NW_ERR_REPORT_ERROR
```

Comment:
You should add these at ```#else``` case also.

Reply:
Added the macros. Thanks for spotting this.